### PR TITLE
Review page

### PR DESCRIPTION
**Pull request by vlad0**
Add review page and all relevant components.

### CODE REVIEW

File: src/client/app/review/product-items/product-items.component.ts [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+@Component({
+  moduleId: module.id,
+  selector: 'product-items',
+  templateUrl: 'product-items.component.html'
+})
+export class ProductItemsComponent implements OnChanges, OnInit {
+  @Input() productList:ProductList;
+
+  term = new FormControl();
```
Comment (Elinos) [inline src/client/app/review/product-items/product-items.component.ts]: What is "term"?

File: src/client/app/review/product-items/product-items.component.ts [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+          })
+
+          return category;
+        });
+      });
+  }
+
+  ngOnChanges(changes:{[propKey:string]:SimpleChange}) {
+    for (let propName in changes) {
+      this[propName] = changes[propName].currentValue;
```
Comment (Elinos) [inline src/client/app/review/product-items/product-items.component.ts]: What is this doing? Maybe too generic names?

File: src/client/app/review/review.component.ts [hunk 3]
```diff
@@ -1,22 +1,34 @@
...
   selector: 'review',
   templateUrl: 'review.component.html',
-  providers: [MenuService]
+  providers: [ProductListService]
 })
 
 export class ReviewComponent implements OnInit {
-  constructor(private menuService:MenuService) {
+  productList:ProductList
+  test:string = "My test";
```
Comment (Elinos) [inline src/client/app/review/review.component.ts]: Do we need this?

File: src/client/app/shared/models/product-category.ts [hunk 4]
```diff
@@ -0,0 +1,7 @@
+import {ProductListItem} from "./product-list-item";
+export class ProductCategory {
+  public cachedItems:ProductListItem[];
+
+  constructor(public title:string, public items:ProductListItem[]) {
```
Comment (Elinos) [inline src/client/app/shared/models/product-category.ts]: Do we need the fields to be public?

File: src/client/app/shared/models/product-list-item.ts [hunk 5]
```diff
@@ -0,0 +1,6 @@
+import {ProductCategory} from "./product-category";
+
+export class ProductListItem {
+  constructor(public title:string, public description:string) {
```
Comment (Elinos) [inline src/client/app/shared/models/product-list-item.ts]: Do we need the fields to be public?

**Comment by vlad0** [comment]
too early for merge in develop.

**Pull request closed without merging**